node/resolvers/Mutations/Users.ts

```diff
@@ -38,6 +38,25 @@ const addUserToMasterdata = async ({ masterdata, params }: any) => {
   return newUser.DocumentId
 }
 
+const getUser = async ({ masterdata, params }: any) => {
+  return masterdata
+    .searchDocuments({
+      dataEntity: 'CL',
```

Comment:
```suggestion
      dataEntity: config.name,
```
CL is the profile system; it's okay if a user already has an entry there. The app needs to check the storefront-permissions MD table instead.